web/components/TopBar/ResetButton.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <v-dialog v-model="data.dialog" width="500">
+    <template #activator="{ on }">
+      <v-btn class="ma-5 mb-0" color="error" v-on="on"> Reset </v-btn>
```

Comment:
not aligned.

![スクリーンショット 2022-02-28 8 37 55](https://user-images.githubusercontent.com/44139130/155904888-6d06c080-12dd-4529-a374-b5d34fcaaa0a.png)

```suggestion
      <v-btn class="ma-2" color="error" v-on="on"> Reset </v-btn>
```